db/cardEntry.js

```diff
@@ -0,0 +1,105 @@
+// Functions for manipulating the card_entry table.
```

Comment:
Great job with this! Will definitely make the next issues much easier now that we have a good interface for accessing the database!